src/util.ts

```diff
@@ -1,5 +1,23 @@
 export const isClient = typeof window === 'object';
 
-export const on = (obj: any, ...args: any[]) => obj.addEventListener(...args);
+export const on = (obj: any, ...args: any[]) => {
+  try {
+    obj.addEventListener(...args);
+  } catch (error) {
+    handleError(error);
+  }
+};
 
-export const off = (obj: any, ...args: any[]) => obj.removeEventListener(...args);
+export const off = (obj: any, ...args: any[]) => {
+  try {
+    obj.removeEventListener(...args);
+  } catch (error) {
```

Comment:
I'm not sure we should be swallowing these errors. Also, these utilities are use by other hooks, too, not only by the `useMediaDevices` hook, if we want to swallow the errors, would be better to do it in the `useMediaDevice` hook directly.

Reply:
Well, I don't see why `useMediaDevices` should have a different behavior, especially, if the source of the issue is the same.
As we could have similar issues for another `addEventListener` or `removeEventListener`, I thought we'd rather handle it globally and swallow the errors in PROD env (unless I misunderstood the issue) ?

Reply:
I don't think we should be swallowing these errors. You are swallowing the errors in all environments. In PROD you are not logging.

Reply:
Ah! I see you point (I interpreted swallowing as not logging).
I clearly misunderstood what was expected in the issue, how would you approach it ?
Doing it only for the `useMediaDevices` hook or just checking if the `obj` is defined before manipulating the event listeners ?

Reply:
Yes, checking if object exists before manipulating it sounds good.

Reply:
Done in a new commit (don't know what you are following in the project, you rather update the commit or add new commit for feedback ?)